penguin/web/app.py

```
@@ -278,16 +284,32 @@ async def read_root():
             """Serve the main web UI."""
             index_path = static_dir / "index.html"
             if index_path.exists():
-                return FileResponse(str(index_path))
+                return FileResponse(str(index_path), headers=NO_CACHE_HEADERS)
             return {"message": "Penguin API is running. Web UI not found."}
 
+        @app.get("/chat")
+        async def read_chat():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `read_chat`

(ANN202)